packages/tools/devtools/devtools-example/src/FluidObject.ts

```diff
@@ -8,15 +8,9 @@ import { SharedCell } from "@fluidframework/cell";
 import { type IFluidHandle, type IFluidLoadable } from "@fluidframework/core-interfaces";
 import { SharedCounter } from "@fluidframework/counter";
 import { type IFluidDataStoreRuntime } from "@fluidframework/datastore-definitions";
-import { type SharedObjectClass } from "@fluidframework/fluid-static";
```

Comment:
SharedObjectClass was imported due to a workaround that is no longer necessary.  (See below).